tests/frontend/test_facilities.py

```diff
@@ -51,3 +51,8 @@ def test_0080_building_level_rooms(self):
             "/facilities/buildings/{}/levels/{}/rooms/".format(
                 self.building.id, self.room.level),
             "facilities/rooms.html")
+
+    def test_0090_overcrowded_rooms(self):
```

Comment:
Great that you provided a frontend test!

Minor detail, please remove the number from your new test case.
I know a lot of tests contain these, but including a number to enforce an execution order is pointless and unidiomatic.
Most of the tests have been left that way, but don't let that confuse you.